dex-auth/keycloak-gatekeeper/base/virtualservice.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: keycloak-gatekeeper
+spec:
+  gateways:
+  - kubeflow/kubeflow-gateway
```

Comment:
Is the `kubeflow/kubeflow-gateway` supposed to refer the `kubeflow-gateway` in the `kubeflow` namespace?

Reply:
Yes this is kubeflow namespace's kubeflow gateway. The naming style is namespace/gateway if the gateway referred to is not in the same namespace.
https://github.com/istio/istio/issues/11131